src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1179,6 +1179,13 @@ await UploadBlobAsync(NodeTaskRunnerStartTaskFileName, SerializeNodeTask(uploadS
 
             return cloudTask;
 
+            static string AppendPathToUrl(string url, string path)
```

Comment:
something like this will ensure / is not a problem:

```
static string AppendPathToUrl(string url, string path)
{
    var uri = new UriBuilder(url);
    string existingPath = uri.Path.TrimEnd('/');
    string appendedPath = path.TrimStart('/');
    uri.Path = string.Concat(existingPath, "/", appendedPath);
    return uri.Uri.ToString();
}
```

Reply:
Done